src/com/booksaw/betterTeams/commands/teama/TeleportTeama.java

```diff
@@ -0,0 +1,113 @@
+package com.booksaw.betterTeams.commands.teama;
+
+import com.booksaw.betterTeams.CommandResponse;
+import com.booksaw.betterTeams.Team;
+import com.booksaw.betterTeams.commands.presets.TeamSelectSubCommand;
+import com.booksaw.betterTeams.message.HelpMessage;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+public class TeleportTeama extends TeamSelectSubCommand {
+
+    @Override
+    public CommandResponse onCommand(CommandSender sender, String label, String[] args, Team team) {
+        Team specifiedTeam = Team.getTeam(args[0]);
+
+        if (specifiedTeam == null) {
+            return new CommandResponse("noTeam");
+        }
+
+        if (!(sender instanceof Player)) {
```

Comment:
There is a method to remove this if statement

The method: https://github.com/booksaw/BetterTeams/blob/5fb9fd6cf92c81d3f4660b0cf866f351b7d05762/src/com/booksaw/betterTeams/commands/SubCommand.java#L107
Example of the method: https://github.com/booksaw/BetterTeams/blob/5fb9fd6cf92c81d3f4660b0cf866f351b7d05762/src/com/booksaw/betterTeams/commands/presets/TeamSubCommand.java#L54